CONTRIBUTING.md

```diff
@@ -12,6 +12,21 @@ you planning to submit your own fix in a pull request?
 [issues]: https://github.com/cfe-lab/MiCall/issues
 
 ## Development
+
+The easiest way to start developing MiCall is by using DevContainers.
+
+1. **Open Project**: Open the MiCall project folder in your editor, which should prompt to "Reopen in Container" or similar — click this to initialize the DevContainer environment.
```

Comment:
Please mention the VS Code [dev containers extension] and the option to run in GitHub code spaces.

[dev containers extension]: https://marketplace.visualstudio.com/items?itemName=ms-vscode-remote.remote-containers

Reply:
Done